cmd/flag.go

```diff
@@ -31,9 +32,14 @@ import (
 var log = vklogv2.New(nil)
 
 func installFlags(flags *pflag.FlagSet, c *provider.Opts) {
+	hostname, _ := os.Hostname()
+	if hostname == "" {
+		hostname = "systemk"
+	}
+
 	flags.StringVar(&c.KubeConfigPath, "kubeconfig", "", "cluster client configuration")
 	flags.StringVar(&c.KubeClusterDomain, "cluster-domain", provider.DefaultKubeClusterDomain, "cluster domain")
-	flags.StringVar(&c.NodeName, "nodename", "systemk", "value to be set as the Node name and label node.k8s.io/hostname")
+	flags.StringVar(&c.NodeName, "nodename", hostname, "value to be set as the Node name and label node.k8s.io/hostname")
```

Comment:
I would rather have this set to empty string so that the fallback code kicks in https://github.com/virtual-kubelet/systemk/blob/a1aa0c1552ff44ba6e9a3ed341eabf4d08359aa8/internal/provider/opts.go#L117-L120

Reply:
will do

Reply:
Thank you!